core/controllers/contributor_dashboard.py

```diff
@@ -292,7 +302,10 @@ def _get_reviewable_exploration_opportunity_summaries(
             suggestion.target_id
             for suggestion in
             suggestion_services.get_suggestions_with_translatable_explorations(
-                in_review_suggestions)
+                [in_review_suggestion
```

Comment:
I think we should instead modify the `suggestion_services.get_reviewable_translation_suggestions_by_offset` call above to accept a language code and perform the filtering at the storage layer if applicable so that we don't need to load suggestions for other languages.

Reply:
Done, please take a look at the latest push